modules/runtime/pages/purge-tool-changelog.adoc

```diff
@@ -6,6 +6,40 @@ This is the changelog of the purge tool.
 
 The purge tool is used to remove data from bonita archive tables. It is useful for big production environments.
 
+== 2.0.0 - September 11, 2024
```

Comment:
@rbioteau @pabloalonso  I guess we should change this date... any idea when the release is scheduled?